components/automate-openjdk/habitat/plan.sh

```diff
@@ -8,10 +8,10 @@ pkg_origin=chef
 pkg_maintainer="Chef Software Inc. <support@chef.io>"
 pkg_name=automate-openjdk
 # Release archive https://jdk.java.net/archive/
-pkg_version=11.0.10+9
-pkg_source=https://github.com/AdoptOpenJDK/openjdk11-binaries/releases/download/jdk-11.0.10%2B9/OpenJDK11U-jdk_x64_linux_hotspot_11.0.10_9.tar.gz
-pkg_shasum=ae78aa45f84642545c01e8ef786dfd700d2226f8b12881c844d6a1f71789cb99
-pkg_filename=OpenJDK11U-jdk_x64_linux_hotspot_11.0.10_9.tar.gz
+pkg_version=11.0.12+7
+pkg_source=https://github.com/adoptium/temurin11-binaries/releases/download/jdk-11.0.12%2B7/OpenJDK11U-jdk_x64_linux_hotspot_11.0.12_7.tar.gz
```

Comment:
Is there a reason that automate uses the full JDK vs. just the JRE which is all you need for Elasticsearch? This would reduce the installer size by 150 megs.

Reply:
@kalroy @vivek-yadav any insights? I can change it in this PR, although I'm not sure if there was a reason to use JDK instead of JRE here.